fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -130,4 +160,17 @@ public String apply(final Integer type) {
 
     private static final Logger LOGGER = getLogger(DefaultMessageFactory.class);
 
+    private String getPropertiesAsString(final FixityEvent fixityEvent) {
+        final StringBuilder propsStrBuilder = new StringBuilder();
+        for (Map.Entry<String,String> entry : fixityEvent.getInfo().entrySet()) {
+            propsStrBuilder.append(entry.getKey()).append("->").append(entry.getValue()).append(",");
```

Comment:
@ajs6f I don't think you can have multiple headers of the same name; I believe the second instance will overwrite the first -- certainly that's the case for `STOMP`, and I'd think it is for `openwire` as well. If there are two applicable conditions, using a comma delimited list on a single header would follow existing practice.